strapi/strapi
# Fix count issue when deep filtering

### What does it do?

Changes the count implementation in the query builder, so it uses `countDistinct()` based on the column id

### Why is it needed?

When counting with deep filters columns were counted multiple times based on their relation

### How to test it?

Create a content type with a relation (one-to-many or many-to-many) (you can take a look at the issue linked below for more details) add multiple elements to the relation field and then try to get the content type while filtering on that field, the total in the meta should be correct.

### Related issue(s)/PR(s)

fix #7631
